src/main/java/seedu/tinner/logic/parser/EditCommandParser.java

```diff
@@ -0,0 +1,59 @@
+package seedu.tinner.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.tinner.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.tinner.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.tinner.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.tinner.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.tinner.logic.parser.CliSyntax.PREFIX_PHONE;
+
+import seedu.tinner.commons.core.index.Index;
+import seedu.tinner.logic.commands.EditCompanyCommand;
+import seedu.tinner.logic.commands.EditCompanyCommand.EditCompanyDescriptor;
+import seedu.tinner.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new EditCommand object
+ */
+public class EditCommandParser implements Parser<EditCompanyCommand> {
```

Comment:
Should this class be called EditCompanyCommandParser instead?

Reply:
Yea, it definitely should. Thanks for pointing this out!